ui-tests/test/smoke.spec.ts

```diff
@@ -31,19 +31,22 @@ test.describe('Smoke', () => {
     ]);
 
     // Enter code in the first cell
-    await notebook.fill('//textarea', 'import math');
-    await notebook.press('//textarea', 'Enter');
-    await notebook.press('//textarea', 'Enter');
-    await notebook.fill('//textarea', 'math.pi');
+    await notebook.locator(
+      '.jp-Cell-inputArea >> .cm-editor >> .cm-content[contenteditable="true"]'
+    ).type(`import math
+
+math.pi`);
 
     // Run the cell
     runAndAdvance(notebook);
 
     // Enter code in the next cell
-    await notebook.fill(
-      "//div[normalize-space(.)=' ​']/div[1]/textarea",
-      'import this'
-    );
+    await notebook
+      .locator(
+        '.jp-Cell-inputArea >> .cm-editor >> .cm-content[contenteditable="true"]'
```

Comment:
This seems to be hanging on the Chrome UI tests (not firefox), need to investigate why. Maybe there is the kernel selection dialog grabbing the focus.